packages/ubuntu_desktop_installer/lib/installer.dart

```diff
@@ -200,17 +194,34 @@ class _UbuntuDesktopInstallerWizardState
         ),
         Routes.selectGuidedStorage: WizardRoute(
           builder: SelectGuidedStoragePage.create,
-          onNext: (settings) => Routes.writeChangesToDisk,
+          onNext: (_) =>
+              (!model.hasEncryption) ? Routes.writeChangesToDisk : null,
```

Comment:
Minor style nit: other such conditionals in this file do not enclose the negated attribute in parentheses.

Reply:
Good spot, thanks. One more character and it would fit on one line. :) I wanted to have it negated to be able to organize the routes so that the conditions can be "if something, skip to route" and it jumps over any routes in between. The null fallback lets it continue to the next route regardless of what it is.

Reply:
Yes, that looks good, and I find it quite readable.